checkout/payment-menu.md

```diff
@@ -0,0 +1,419 @@
+        "language": "sv-SE",
+        "generateRecurrenceToken": false,
+        "urls": {
+            "hostUrls": ["https://example.com", "https://example.net"],
+            "completeUrl": "https://example.com/payment-completed",
+            "cancelUrl": "https://example.com/payment-canceled",
+            "paymentUrl": "https://example.com/perform-payment",
+            "callbackUrl": "https://api.example.com/payment-callback",
+            "termsOfServiceUrl": "https://example.com/termsandconditoons.pdf"
+        },
+        "payeeInfo": {
+            "payeeId": "12345678-1234-1234-1234-123456789012",
+            "payeeReference": "AB832",
+            "payeeName": "Merchant1",
+            "productCategory": "A123",
+            "orderReference" : "or-123456"
+        },
+        "payer": {
+             "consumerProfileRef": "7d5788219e5bc43350e75ac633e0480ab30ad20f96797a12b96e54da869714c4"
+        },
+        "orderItems": [
+            {
+                "reference": "P1",
+                "name": "Product1",
+                "type": "PRODUCT",
+                "class": "ProductGroup1",
+                "itemUrl": "https://example.com/products/123",
+                "imageUrl": "https://example.com/product123.jpg",
+                "description": "Product 1 description",
+                "discountDescription": "Volume discount",
+                "quantity": 4,
+                "quantityUnit": "pcs",
+                "unitPrice": 300,
+                "discountPrice": 200,
+                "vatPercent": 2500,
+                "amount": 1000,
+                "vatAmount": 250
+            },
+            {
+                "reference": "P2",
+                "name": "Product2",
+                "type": "PRODUCT",
+                "class": "ProductGroup1",
+                "description": "Product 2 description",
+                "quantity": 1,
+                "quantityUnit": "pcs",
+                "unitPrice": 500,
+                "vatPercent": 2500,
+                "amount": 500,
+                "vatAmount": 125
+            }
+        ],
+        "riskIndicator" : {
+            "deliveryEmailAddress" : "string",
+            "deliveryTimeFrameIndicator" : "01",
+            "preOrderDate" : "YYYYMMDD",
+            "preOrderPurchaseIndicator" : "01",
+            "shipIndicator" : "01",
+            "giftCardPurchase" : false,
+            "reOrderPurchaseIndicator" : "01",
+            "pickUpAddress" : {
+                "name" : "companyname",
+                "streetAddress" : "string",
+                "coAddress" : "string",
+                "city" : "string",
+                "zipCode" : "string",
+                "countryCode" : "string"
+            }
+        },
+    }
+}
+```
+
+{:.table .table-striped}
+| ✔︎︎︎︎︎ (Required) | Property                              | Type         | Description       |
+|:-:|:--------------------------------------|:-------------|:------------------|
```

Comment:
```suggestion
| :---------------: | :------------------------------------ | :----------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```
This table should be formatted like other tables in the project. Using the `yzhang.markdown-all-in-one` extension you can use the auto-formatter for the table to do it for you.

Reply:
Seems like a great plugin, but the shortcut to format the table doesn't work on Mac. It also doesn't seem to be available inthe command pallette, so I have no way to invoke the table formatting.

Reply:
Hmm, it formats on save for me. I do not know why.
Try the `Format document` command? `Shift`+`Alt`+`F` for me. Search for it in the command pallette.

Reply:
Ah, so it's not a specific command to the extension, it's plugged in to the global format command? Scary and neat.